deliveries/bluesky_milestone_1.md

```diff
@@ -0,0 +1,44 @@
+
+[Bluesky project]()
+
+**Context** (optional)
+> Please provide a short paragraph or two connecting the deliverables in this milestone and describing their purpose.
+
+In this [technical showcase](https://medium.com/@opensmartcontract/learn-ink-by-example-order-food-on-blockchain-a4024b2dee4a) we implemented with ink! smart contract language including custom macros, security, testing scripts, deployment and documents. It delivers the first milestone: define a specific use case: Order Food on Blockchain, implement in ink! with macros encapsulating reusable business logic, tested and deployed, then publish article to share with the community learning experiences / resources / recommendations about ink! and openBrush. 
+
+**Deliverables**
+> Please provide a list of all deliverables of the milestone extracted from the initial application and a link to the deliverable itself. Ideally all links inside the below table should include a commit hash, which will be used for testing. If you don't provide a commit hash, we will work off the default branch of your repository. Thus, if you plan on continuing work after delivery, we suggest you create a separate branch for either the delivery or your continuing work. 
+> 
+> If there is anything particular about any of the deliverables we or a future reader should know, use the respective `Notes` column.
+
+| Number | Deliverable | Link | Notes |
+| ------------- | ------------- | ------------- |------------- |
+| 1. | Define use case | [Food Order on Blockchain](https://medium.com/@opensmartcontract/learn-ink-by-example-order-food-on-blockchain-a4024b2dee4a) | define a contrived use case with [persona](https://docs.google.com/document/d/1yiZrPnSaz9myrXOrR2k2IALbJA9XcjeEJ-wCmhn3S3o/edit#heading=h.s7qu46re6cic), [workflow](https://docs.google.com/document/d/1yiZrPnSaz9myrXOrR2k2IALbJA9XcjeEJ-wCmhn3S3o/edit#heading=h.p52xmq7d7g3m), [target audiences](https://docs.google.com/document/d/1yiZrPnSaz9myrXOrR2k2IALbJA9XcjeEJ-wCmhn3S3o/edit#heading=h.y8aexxoud7oj) for ink! smart contract implementation |
+| 2. | Investigate OpenZepplin's's security / ownership to see how it might fit | [OpenBrush vs openzeppelin on Ownable and AccessControl](https://docs.google.com/document/d/1yiZrPnSaz9myrXOrR2k2IALbJA9XcjeEJ-wCmhn3S3o/edit#heading=h.ahhyroiqktdx) | Implemented ownable for security and restaurants/coutier roles for access control |
+| 3. | Define list of ink! Macros | Define declarative and procedure macros | 1) [Payment_macro](https://github.com/InkSmartContract/foodorder-smartcontract/blob/main/contracts/foodorder/logic/helpers/helpers.rs) as declarative macro; 2) [crud_macro](https://github.com/InkSmartContract/foodorder-smartcontract/tree/main/contracts/foodorder/crud-macro) as procedure macro encapsulating reusable logic to simplify code |
+| 4. | Smart contracts | Implemented in ink! smart contract with macros] | [Github open source repository](https://github.com/InkSmartContract/foodorder-smartcontract.git) |
```

Comment:
Can we think of a better name for [this](https://github.com/InkSmartContract/foodorder-smartcontract/blob/93af9fe693a5937dd15550594605c24990ebc9d6/contracts/foodorder/src/lib.rs#L5) contract?

Reply:
All public messages should be documented. e.g. [Here](https://github.com/InkSmartContract/foodorder-smartcontract/blob/93af9fe693a5937dd15550594605c24990ebc9d6/contracts/foodorder/logic/traits/courier_service.rs#L5)

Reply:
I'll let some of the other curators / devs weigh in on the Rust code

Reply:
Documented public messages: [commit1](https://github.com/InkSmartContract/foodorder-smartcontract/commit/b91013a8ec4066a056a31359dda6e7cba2e2fc30) and [commit2](https://github.com/InkSmartContract/foodorder-smartcontract/commit/58ac50885cc97bebad89e3a62f5c2f111fff5e50)

Reply:
Renamed the contract to BlockchainFoodOrder. Documented all public messages. Refreshed README.md accordingly.